src/Controller.js

```diff
@@ -0,0 +1,125 @@
+(function exportController() {
+    class Controller {
+        constructor(ship) {
+            this.ship = ship;
+            this.initialiseSea();
+            this.waterState = false; // here false represents water0.png
+            
+            document.querySelector('#sailbutton').addEventListener('click', () => {
+                this.setSail();
+            })
+
+            this.renderHeadsUp();
+        }
+
+        initialiseSea() {
+            window.setInterval(this.changeWater, 1000)
+        }
+
+        changeWater() {
+            const viewport = document.querySelector('#viewport');
+
+            if(!this.waterState) {
+                viewport.style.backgroundImage = "url(../images/water1.png)";
+                this.waterState = true;
+            } else {
+                viewport.style.backgroundImage = "url(../images/water0.png)";
+                this.waterState = false;
+            }
```

Comment:
You could use a ternary for this and only call the change to water state once with a toggle:
```
            !this.waterState
                ? (viewport.style.backgroundImage = "url(../images/water1.png)")
                : (viewport.style.backgroundImage = "url(../images/water0.png)");

            this.waterState = !this.waterState;
```